docs/source/config_file.rst

```diff
@@ -3,17 +3,22 @@
 The mypy configuration file
 ===========================
 
-Mypy supports reading configuration settings from a file.  By default
-it uses the file ``mypy.ini`` with a fallback to ``.mypy.ini``, then ``pyproject.toml``,
-then ``setup.cfg`` in the current directory, then ``$XDG_CONFIG_HOME/mypy/config``, then
-``~/.config/mypy/config``, and finally ``.mypy.ini`` in the user home directory
-if none of them are found; the :option:`--config-file <mypy --config-file>` command-line flag can be used
-to read a different file instead (see :ref:`config-file-flag`).
+Mypy supports reading configuration settings from a file with the following precedence order:
+
+    1. The file passed into the :option:`--config-file <mypy --config-file>` command-line flag (see :ref:`config-file-flag`)
+    2. ``mypy.ini``
+    3. ``.mypy.ini``
+    4. ``pyproject.toml``
+    5. ``setup.cfg`` in the current working directory
```

Comment:
I think the "in the current working directory" in the original was meant to apply to all of them

Reply:
I think so too---I meant to imply that 2-5 would be in the current working directory but I don't see a good way to explicitly say that, do you have any suggestions on how to make it easy for users to read? Or maybe it doesn't even need to be mentioned at all?

Reply:
Updated in commit [ad59554](https://github.com/python/mypy/pull/15234/commits/ad595548812722795b29a2bed047ce09c57ffa85)

I changed "in the current working directory" to placing ```./``` before each of the files, since that indicates the current working directory.